transformers/payment.go

```diff
@@ -66,3 +68,55 @@ func (pt PaymentTransformer) TransformToRest(dbResource models.PaymentResourceDB
 
 	return paymentResource
 }
+
+func getRefundsDB(refunds []models.RefundResourceRest) []models.RefundResourceDB {
```

Comment:
Can you add unit test coverage for these new methods?

Reply:
Done!